raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -37,6 +38,28 @@ library ChannelManagerLibrary {
     /// @return channel (address) the address of the NettingChannelContract of the two parties.
     function getChannelWith(Data storage self, address caller_address, address partner)
         constant
+        returns (address, bool)
+    {
+        bytes32 party_hash = partyHash(caller_address, partner);
+        if (self.channel_addresses[party_hash] == 0x0 || 
+            !contractExists(self, self.channel_addresses[party_hash])) {
```

Comment:
Question here: If there is a channel in the mapping but the contract for that channel does not exist, something wrong must have happened, right? When would that case occur? If so, should we not throw?